lib/filter/filter-expr-grammar.ym

```diff
@@ -103,12 +105,11 @@ filter_simple_expr
 	| KW_NETMASK '(' string ')'    { $$ = filter_netmask_new($3); free($3); }
   | KW_NETMASK6 '(' string ')'   {
   #if ENABLE_IPV6
-                                    $$ = filter_netmask6_new($3);
-  #else
-                                    YYERROR;
+    $$ = filter_netmask6_new($3);
```

Comment:
What happens when syslog-ng is configured without IPv6 support but the user tries to use this filter?